src/PIL/JpegImagePlugin.py

```diff
@@ -432,7 +433,8 @@ def draft(self, mode, size):
         self.tile = [(d, e, o, a)]
         self.decoderconfig = (scale, 0)
 
-        return self
```

Comment:
This is breaking change. Previously, `draft()` could return `None` or `self`. This is very strange behavior, I don't think anyone actually used it. Now `draft()` returns the results of its work or `None`.

I'll add this in docs.